### PR TITLE
Memory leak fixes

### PR DESCRIPTION
**Pull request by megheaiulian**
Memory leak fixes

### CODE REVIEW

File: cosmoz-omnitable-repeater-behavior.html [hunk 1]
```diff
@@ -257,6 +258,15 @@
 			if (instance._flushProperties) {
 				instance._flushProperties(true);
 			}
+		},
+
+		detached() {
+			if (this._elements) {
```
Comment (nomego) [inline cosmoz-omnitable-repeater-behavior.html]: Why shouldn't we splice columns if we have elements?

File: cosmoz-omnitable-templatizer.html [hunk 2]
```diff
@@ -116,6 +116,57 @@
...
+
+		release() {
+			const instances = this._templateInstancesInUse.splice(0)
+				.concat(this._reusableTemplateInstances.splice(0));
+
+			if (IS_V2) {
+				return;
+			}
+
+			let doV1Release = instance => {
```
Comment (nomego) [inline cosmoz-omnitable-templatizer.html]: const

File: cosmoz-omnitable-templatizer.html [hunk 3]
```diff
@@ -116,6 +116,57 @@
...
+				events.forEach(e => host._unlisten(e.node, e.eventName, listener));
+			};
+
+			instances.forEach(doV1Release);
+		},
+
+		_listenImpl: function (node, eventName, methodName) {
+			this._decoratedEvents = this._decoratedEvents || [];
+
+			let model = this,
```
Comment (nomego) [inline cosmoz-omnitable-templatizer.html]: const

File: cosmoz-omnitable.js [hunk 4]
```diff
@@ -288,6 +288,17 @@
 			this.cancelDebouncer('updateColumns');
 			this.cancelDebouncer('filterItems');
 			this.cancelDebouncer('sortItems');
+
+			this._debouncers = {};
+			this.columns.forEach(column => column.release());
+
+			let listeners = this._decoratedListeners;
```
Comment (nomego) [inline cosmoz-omnitable.js]: const

**Comment by megheaiulian** [comment]
Closing as we'll handle this in 2.x if needed.

**Comment by megheaiulian** [comment]
I know we discussed to not merge this but it includes a ES6 rewrite of the `_removeElements` function `cosmoz-omnitable-repeater` behavior and removing references to `__instance` and `__column` should speed up GC.

**Pull request merged**